mbuild/lattice.py

```diff
@@ -522,4 +524,8 @@ def populate(self, compound_dict=None, x=1, y=1, z=1):
                                     .format(key_id, err_type))
         # set periodicity
         ret_lattice.periodicity = np.asarray([a * x, b * y, c * z], dtype=np.float64)
+        warn('Periodicity of non-cubic lattices are not valid with '
```

Comment:
Is this true for non-cubic or non-orthogonal?

Reply:
Thinking over it, periodicity should be ok for all rectangular based unit cells. As long as the angles are 90,90,90. 

I'll change the phrasing for that.